### ISSUE TITLE
[Accessibility, a11y] Screen reader isn't reading quick pick title field

### ISSUE THREAD

**Issue by VSCodeTriageBot** [JacksonKearl/testissues#50940]
**VS Code Version**:  1.67.1

**Operating System**:  Windows 11

**Screen Reader**:  NVDA

**Extensions Installed**:
Put together an extension to quickly demo the bare VS Code UI features without our normal SDK and extension wrappers.
"a11y-kit" (https://marketplace.visualstudio.com/items?itemName=MicroFish.accessibility-testkit)

**Repro Steps**:
  1. Install a11y-kit from VS Code Marketplace
  2. Open command palette and run a11y-kit's menu command
  3. Next, select: "Quick Picks: Show Quick Picks - Single" and observe.

**Expected**:
When entering the quick pick selection screen, the title should be read by the screen reader so that the user understands the context of the selection environment.

**Actual**:
Certain other descriptors are picked up and read, but not the "Quick Pick Placeholder Title".

Note:  A user posted an issue on one of our extensions explaining that the title is picked up on JAWS but not NVDA.  I have only verified this on VoiceOver and NVDA - neither screen reader reads the title as far as I can tell.

**Attachments**:
![sqp_macos](https://user-images.githubusercontent.com/40250218/168152938-07b1d627-7861-435e-abd5-a05e9d9faf30.gif)

Thank you
